intuita-webview/src/shared/Snippet/calculateDiff.tsx

```diff
@@ -0,0 +1,32 @@
+import { monaco } from './DiffEditor';
+
+export type Diff = { added: number; removed: number };
+
+export const getDiff = (lineChanges: monaco.editor.ILineChange[]): Diff => {
+	const diff = new Map<keyof Diff, number>();
```

Comment:
hmm could you create an `added` and `removed` numbers for readability? Since we have two properties, the `map` seem not to be needed.